library/Cargo.toml

```diff
@@ -38,8 +38,10 @@ adler2.debug = 0
 gimli.debug = 0
 gimli.opt-level = "s"
 miniz_oxide.debug = 0
+miniz_oxide.opt-level = "s"
 object.debug = 0
```

Comment:
It would also be interesting to see it's effect on the `object` crate.

Reply:
The effect was either negligible or an increase, so I didn't include that